flights/backend/controllers/flight_controller.go

```diff
@@ -42,14 +42,7 @@ func CreateFlight(ctx *gin.Context) {
 func DeleteFlight(ctx *gin.Context) {
 	httpGin := http.Gin{Context: ctx}
 	var id string
-
-	if err := ctx.BindJSON(&id); err != nil {
```

Comment:
Why delete the error handling part though? Did you forget to undelete it after debugging?